src/xo-app/vm/tab-disks.js

```diff
@@ -1,17 +1,16 @@
-import _, { messages } from 'intl'
 import ActionButton from 'action-button'
-import ActionRowButton from 'action-row-button'
 import Component from 'base-component'
 import HTML5Backend from 'react-dnd-html5-backend'
 import Icon from 'icon'
 import IsoDevice from 'iso-device'
-import Link from 'link'
 import propTypes from 'prop-types-decorator'
 import React from 'react'
 import SingleLineRow from 'single-line-row'
+import SortedTable from 'sorted-table'
 import StateButton from 'state-button'
 import TabButton from 'tab-button'
-import Tooltip from 'tooltip'
+import Link from 'link'
+import _, { messages } from 'intl'
```

Comment:
Sort imports:
1. Default imports sorted alphabetically
2. Named imports not necessarily sorted
3. Local imports

```js
import Aaa from 'aaa'
import Bbb from 'bbb'

import { Foo, Bar } from 'foobar'
import { Baz } from 'baz'

import myStuff from './my-stuff'
```